spec/controllers/news_item_controller_spec.rb

```diff
@@ -2,4 +2,26 @@
 
 RSpec.describe NewsItemsController, type: :controller do
 
+  context 'when accessing a published news_item' do
+    let(:published_news_item) { create :news_item, state: 'published' }
+
+    it 'returns a 200 status code' do
+      get :show, id: published_news_item
+      expect(response).to have_http_status(200)
+    end
+  end
+
+  context 'when accessing a non-published news_item' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.